commands/RamseteCommand.java

```diff
@@ -162,9 +162,9 @@ public void execute() {
       rightOutput = rightSpeedSetpoint;
     }
 
-    // Logger.consoleLog("LeftspeedSetPoint: %s, Left Speed: %s, LeftOutput: %s, rightSpeedSetPoint: %s Right Speed: %s rightOutput: %s", leftSpeedSetpoint, m_speeds.get().leftMetersPerSecond, leftOutput, rightSpeedSetpoint, m_speeds.get().rightMetersPerSecond, rightOutput);
-    // Logger.consoleLog("Trajectory %s", m_trajectory.sample(curTime));
-    // Logger.consoleLog("Pose: %s", m_pose.get().toString());
+    ConsoleLogger.consoleLog("LeftspeedSetPoint: %s, Left Speed: %s, LeftOutput: %s, rightSpeedSetPoint: %s Right Speed: %s rightOutput: %s", leftSpeedSetpoint, m_speeds.get().leftMetersPerSecond, leftOutput, rightSpeedSetpoint, m_speeds.get().rightMetersPerSecond, rightOutput);
```

Comment:
do we really want to revive these? just seems like noise

separately: we don't use RamsetCommand anymore, do we?

Reply:
bump

Reply:
Yeah, we don't care much about them and we don't use Ramsete Command anymore. I'll leave these here for now and just delete the file in 2024-updated